fluentd/lib/in_tail_metric/in_tail_metric.rb

```diff
@@ -566,6 +566,7 @@ def on_notify
         @line_buffer_timer_flusher.on_notify(self) if @line_buffer_timer_flusher
         @io_handler.on_notify if @io_handler
         @totalbytes_collected=@write_watcher.update_totalbytes_collected(@path)
+        @log.debug "TOTALBYTES LOGGED #{@totalbytes_logged} TOTALBYTESCOLLECTED #{@totalbytes_collected}"
```

Comment:
Remove the @log - this is a sensitive path. I'm not sure what the overhead of disabled logs statements is but best not take a chance.  It works now, so we can get the values via metrics :)

Reply:
resolved